sentry_redmine/plugin.py

```diff
@@ -205,18 +206,31 @@ def get_config(self, project, **kwargs):
                     for p in priorities['issue_priorities']
                 ]
                 self.add_choices('default_priority', tracker_choices, choices_value)
-                
-        if not has_credentials and not kwargs.get('add_additial_fields'):
-            for field_name in ['project_id', 'tracker_id', 'default_priority']:
+
+        if not has_credentials:
+            for field_name in ['project_id', 'tracker_id', 'default_priority', 'extra_fields']:
                 self.remove_field(field_name)
-        
+
         return self.fields
-    
+
 
     def validate_config(self, project, config, actor):
         super(RedminePlugin, self).validate_config(project, config, actor)
+        self.client_errors = []
```

Comment:
A better approach here would be to coerce the fields to a map so you’re not reliant on order

Fields = {f[‘name’]: f for f in self.fields}

You could even just create a list of the required fields already being their config using something similar with list comprehension 

Pardon syntax I’m on my phone

Reply:
refactored the code a little differently, but let me know your thoughts.